packages/astro/src/core/config/schema.ts

```diff
@@ -525,6 +587,12 @@ export const AstroConfigSchema = z.object({
 				.optional()
 				.default(ASTRO_CONFIG_DEFAULTS.experimental.security),
 			i18nDomains: z.boolean().optional().default(ASTRO_CONFIG_DEFAULTS.experimental.i18nDomains),
+			env: z
+				.object({
+					schema: envSchemas.EnvSchema.optional(),
+				})
+				.optional()
+				.default({}),
```

Comment:
nit: for defaults, we usually use `ASTRO_CONFIG_DEFAULTS`

Reply:
Saw that but also saw some config use an empty object when there's actually no default. What do you recommend here?